web/themes/custom/pineapple/sass/components/_card.scss

```diff
@@ -4,6 +4,16 @@
   height: 100%;
   overflow: hidden;
 
+  &:hover,
+  &:focus {
+    cursor: pointer;
+    background-color: darken($light-gray, 10%);
```

Comment:
Reduce this to 5% and increase the opacity to 0.9, the current values are too harsh of a change they only need to be slight

Reply:
done